### ISSUE TITLE
Startaufstellung einer Spielpaarung

### ISSUE THREAD

**Issue by ghin27** [diddipoeler/sportsmanagement#543]
Man kan zwar Startaufstellung anlegen (wie: Torwart, Mittelfeld, Angreifer, Ersatzbank), aber auf der Seite wird (Abwehr, Mittelfeld, Angreifer) angezeigt der "Torwart" nicht.
Die angelegte Ersatzbank Spieler werden auch auf der Seite in Startaufstellung angelegt oder angezeigt und nicht als Ersatzspieler.

**Comment by svdoldie** [comment]
Das mit dem Towart funktioniert inzwischen.
Bei dem zweiten Punkt weiß ich nicht wie das gemeint ist

**Comment by ghin27** [comment]
Ich füge ein Bild unter der Beitrag "Bug oder nicht" da kann ich besser beschreiben

**Comment by svdoldie** [comment]
Ok, jetzt weiß ich was du meinst.
Der Bereich, wo du das eintragen möchtest, ist dafür nicht geeignet. 
Der ist nur für die Spieler, die zu Beginn des Spiels auf dem Platz standen.
Dafür müsste man noch einen zusätzlichen Bereich programmieren,
wo du alle Spieler eintragen kannst, die für das Spiel nominiert worden sind, aber nicht eingesetzt wurden.

**Comment by ghin27** [comment]
...und ich dachte es ist für Ersatzspieler gedacht :).

**Comment by svdoldie** [comment]
Kurze Frage zwischendurch. Sollen die Auswechslungen schon gespeichert werden können?

**Comment by diddipoeler** [comment]
Kannst ja bitte noch einmal testen.

**Comment by svdoldie** [comment]
Geht bei mir weder im Backend, noch im Frontend

**Comment by diddipoeler** [comment]
Bei mir läuft es.

**Comment by svdoldie** [comment]
ja, das hab ich gesehen. Ich frage mich nur, warum das bei mir auf keiner Seite geht

**Comment by diddipoeler** [comment]
Kannst du das mal bitte genauer beschreiben ?
Was nicht funktioniert ?

**Comment by svdoldie** [comment]
Die Eingabe funktioniert soweit. Im Backend erscheint beim speichern dieses ajaxrad und dreht sich unendlich. Im Frontend passiert beim speichern gar nicht. Alles ohne Fehlermeldung

**Comment by diddipoeler** [comment]
Mach mal bitte ein Update der Komponente.
Sende mir mal bitte die Admin Zugangsdaten per Mail zu.

**Comment by svdoldie** [comment]
Die Komponente ist eigentlich immer auf dem neuesten Stand.
Zugangsdaten sind unterwegs

**Comment by svdoldie** [comment]
Noch zwei Fehler bei der Startaufstellung:
 1. Spieler werden entsprechend ihrer Teamposition gespeichert und nicht nach der Position im Spiel.
    z.b. wird ein Mittelfeldspieler wenn er in der Abwehr eingestetzt wird, trotzdem dem Mittelfeld zugeordnet.
2. Innerhalb der Positionen lassen sich die Spieler nicht hoch und runter schieben

**Comment by svdoldie** [comment]
Die Auswechselungen lassen sich im Backend nun speichern :+1:

**Comment by diddipoeler** [comment]
Spieler hoch und runter müsste jetzt funzen.

**Comment by svdoldie** [comment]
jepp , das klappt jetzt :+1:

**Comment by diddipoeler** [comment]
Bitte noch einmal Punkt 1 testen.

**Comment by svdoldie** [comment]
Punkt 1 funktioniert jetzt auch :+1:

**Comment by ghin27** [comment]
Spieler hoch und runter geht, speichern auch aber auswechselung wird nich auf der Webseite angezeigt?
Muss da irgendein Einstellung vorgenommen werden?

**Comment by svdoldie** [comment]
Auf welcher Seite wird das nicht angezeigt? Im Spielbericht funktioniert das?

**Comment by ghin27** [comment]
Ein und ausgewchselte Spieler werden nicht  in Spielbericht angezeigt.

**Comment by diddipoeler** [comment]
Bitte noch einmal testen.

**Comment by ghin27** [comment]
Die auswechselung wird jetzt im Spielbericht angezeigt. Mit der auswechselspieler muss ich noch testen:

**Comment by diddipoeler** [comment]
Kann ich das jetzt schließen ?

**Comment by svdoldie** [comment]
Grundsätzlich funktioniert es, nur der Sinn ist mir nicht ersichtlich.
Wahrscheinlich möchte ghin27 das die nicht eingesetzten Spieler, die im Kader für das Spiel waren,
angezeigt werden. Wenn man dafür eine Position anlegt und die Spieler dort für das jeweilige Spiel zuordnet, wird das als Einsatz gewertet.  Und das stimmt ja eben nicht.
